examples/va/hci/kustomization.yaml

```diff
@@ -7,5 +7,54 @@ components:
   ## It's possible to replace ../../../va/hci/ with a git checkout URL as per:
   ## https://github.com/kubernetes-sigs/kustomize/blob/master/examples/remoteBuild.md
 
+# Add StorageMgmt network template, as it is needed for CephHCI
+patchesJson6902:
```

Comment:
This looks like a good fix to the problem of storage-mgmt being on EDPM nodes only when HCI is used. I have to try it in my env but it looks like the right idea.

Reply:
I've tested in my environment and I see that `kustomize build architecture/examples/va/hci` produces effectively the same yaml in `va2_sriov` branch as the `main` branch. So the change to the HCI VA1 here is fine by me.